### ISSUE TITLE
Certain characters not rendered properly

### ISSUE THREAD

**Issue by majk-p** [qooob/authentic-theme#84]
On Firefox polish specific characters like *ś, ę, ż, ł* are not displayed properly on left menu block. Here's the preview:
![webmin](https://cloud.githubusercontent.com/assets/7260594/6018975/c73e8bbe-ab9b-11e4-86f4-3c4ab3d75336.png)

**Comment by qooob** [comment]
Please try switching to pl.UTF-8 language in Webmin Configuration. In case you have selected user based language in Language and Themes module change it there as well.

It should fix your problem.

**Comment by qooob** [comment]
Did it help?

**Comment by webdo** [comment]
Hi, got the same problem with French language, The UTF-8 worked to fix the webmin part but the virtualmin part still have bugged characters.

**Comment by qooob** [comment]
When you look at the language files from the theme in text editor - do they look correct for you?

**Comment by qooob** [comment]
Alright, I think I'm starting to understand... Do you see Virtualmin in English? Is that what mean?

**Comment by majk-p** [comment]
I already use pl.UTF-8. This may be browser specific, I didn't see the problem occure on Opera browser.

**Comment by majk-p** [comment]
I've worked out something. In my opinion I haven't got necessary fonts installed. I did some experiments with font of `aside` and come up with:

    font-family: "HelveticaNeue-Light","Helvetica Neue Light","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;

as suggested at [css-tricks](http://css-tricks.com/snippets/css/better-helvetica/). After that my menu looks like:

![webmin-new-font](https://cloud.githubusercontent.com/assets/7260594/6050892/996665da-acc3-11e4-8848-385e51913d8c.png)

If it looks fine enough for you I'll make pull request with this change.
